### ISSUE TITLE
npm package requires tslib="^1.10.0"

### ISSUE THREAD

**Issue by tanj** [gmerabishvili/angular-ng-autocomplete#158]
#### Expected result

`npm i angular-ng-autocomplete` should install without errors

#### Actual result

install requires `npm i --force angular-ng-autocomplete`

#### Steps to reproduce

1. `ng new test-project`
2. `cd test-project`
3. `npm i angular-ng-autocomplete`

#### Your Environment

* Version used: 2.0.5
* NPM Version: 7.9.0

#### `ng --version` 
```
Angular CLI: 11.2.14
Node: 14.16.1
OS: win32 x64

Angular: 11.2.14
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1102.14
@angular-devkit/build-angular   0.1102.14
@angular-devkit/core            11.2.14
@angular-devkit/schematics      11.2.14
@schematics/angular             11.2.14
@schematics/update              0.1102.14
rxjs                            6.6.7
typescript                      4.1.5
```

## Bug may be in packaging
Looking at your source for the library it appears that `tslib` is not a peer dependency, but the actual `package.json` that ends up in node_modules has `tslib`

### `node_modules/angular-ng-autocomplete/package.json`
```json
{
  "name": "angular-ng-autocomplete",
  "version": "2.0.5",
  "description": "Angular autocomplete",
  "keywords": [
    "angular",
    "angular-ng-autocomplete",
    "angular autocomplete",
    "angular-autocomplete",
    "angular-auto-complete",
    "autocomplete-component",
    "ng-autocomplete",
    "auto complete",
    "autocomplete",
    "angular-6",
    "angular-7",
    "angular-8",
    "angular-9",
    "angular-10",
    "angular-11"
  ],
  "author": {
    "name": "Giorgi Merabishvili",
    "email": "gtmerabishvili@gmail.com",
    "url": "https://www.linkedin.com/in/giorgi-merabishvili-3719a2121/"
  },
  "license": "MIT",
  "repository": {
    "type": "git",
    "url": "https://github.com/gmerabishvili/angular-autocomplete"
  },
  "homepage": "https://gmerabishvili.github.io/angular-ng-autocomplete/",
  "peerDependencies": {
    "@angular/common": "*",
    "@angular/core": "*",
    "tslib": "^1.10.0"
  },
  "main": "bundles/angular-ng-autocomplete.umd.js",
  "module": "fesm5/angular-ng-autocomplete.js",
  "es2015": "fesm2015/angular-ng-autocomplete.js",
  "esm5": "esm5/angular-ng-autocomplete.js",
  "esm2015": "esm2015/angular-ng-autocomplete.js",
  "fesm5": "fesm5/angular-ng-autocomplete.js",
  "fesm2015": "fesm2015/angular-ng-autocomplete.js",
  "typings": "angular-ng-autocomplete.d.ts",
  "metadata": "angular-ng-autocomplete.metadata.json",
  "sideEffects": false
}

```

**Comment by goenaga** [comment]
Same here.

**Comment by gmerabishvili** [comment]
@tanj  New version coming soon...

**Comment by gmerabishvili** [comment]
@tanj Resolved, please upgrade to latest 2.0.6 version. Thank you!
